x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/DateUtils.java

```diff
@@ -12,18 +12,23 @@
 
 import java.time.Instant;
 import java.time.LocalDate;
+import java.time.OffsetTime;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
 
 import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE;
 import static java.time.format.DateTimeFormatter.ISO_LOCAL_TIME;
+import static java.time.format.DateTimeFormatter.ISO_TIME;
 
 public final class DateUtils {
 
     public static final ZoneId UTC = ZoneId.of("Z");
     public static final String DATE_PARSE_FORMAT = "epoch_millis";
+    // Not available in Java 8
```

Comment:
Again, what's with this comment? If something is not available how can it be defined?

Reply:
Changing the comment to `// In Java 8 LocalDate.EPOCH is not yet available`, as one might wonder why we don't use the predefined constant and will be a marker to remove it when Java8 compatibility is dropped.